<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewhomework</class>
 <widget class="QMainWindow" name="viewhomework">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1321</width>
      <height>731</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(D:/Teme Proiecte/POO PROIECT/background.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="homeworksWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>541</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: #f7f7f7;
    border-radius: 15px;
    font-family: &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif;
    font-size: 14px;
    color: #444444;
}

QLabel {
    color: #777777;
}

QPushButton {
    background-color: #f2d3bd;
    border: none;
    border-radius: 5px;
    color: #ffffff;
    padding: 8px;
}

QPushButton:hover {
    background-color: #e1b6a1;
}</string>
    </property>
   </widget>
   <widget class="QListWidget" name="studentsWidget">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>130</y>
      <width>541</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: #E8E5E5;
    border-radius: 15px;
    font-family: &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif;
    font-size: 14px;
    color: #444444;
}

QLabel {
    color: #777777;
}

QPushButton {
    background-color: #f2d3bd;
    border: none;
    border-radius: 5px;
    color: #ffffff;
    padding: 8px;
}

QPushButton:hover {
    background-color: #e1b6a1;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="selecthomeworkLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Select a homework assignment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>0</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-family: &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif;
    font-size: 12px;
    color: #444444;
    border-radius: 5px;
    padding: 8px;
    border: 1px solid #cccccc;
    background-color: #f2f2f2;
}

QPushButton::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: #cccccc;
    border-left-style: solid;
    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
    background-color: #ffcc99;
}

QPushButton::down-arrow {
    image: url(path/to/down-arrow.png);
    width: 12px;
    height: 12px;
}</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>480</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #ffcc99;
    border: none;
    border-radius: 5px;
    color: #ffffff;
    padding: 8px;
}

QPushButton:hover {
    background-color: #ffbb77;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
